libcove/lib/common.py

```diff
@@ -79,9 +79,21 @@ def unique_ids(validator, ui, instance, schema, id_names=["id"]):
 
         for non_unique_id in sorted(non_unique_ids):
             if len(id_names) == 1:
-                msg = "Non-unique {} values".format(id_names[0])
+                msg = (
+                    "Non-unique {} values. There are grants in your data with the same Id. "
+                    "Each Id must be unique so the grants can be distinguished from each other. "
```

Comment:
You can't make this change here, as the text refers to grants, which is specific to 360Giving. Instead, you need to match on the `error_id` somewhere in the [360Giving dataquality](https://github.com/ThreeSixtyGiving/dataquality/) codebase. One place to do this is to override the `html_error_msg` filter by adding a custom cove_tags, here's where we do it for OCDS: https://github.com/open-contracting/cove-ocds/blob/739af777d76dc6bb5ea5bf82387aa77b76f5060a/cove_ocds/templatetags/cove_tags.py#L15

Happy to have a chat if this isn't clear, as I'm aware it's not well documented.